ecl/hql/hqlgram.y

```diff
@@ -10104,6 +10150,14 @@ dsOption
     | commonAttribute
     ;
 
+lookupOption
+    : LOOKUP            {   $$.setExpr(createExprAttribute(lookupAtom, createConstant(true)), $1); }
+    | LOOKUP '(' expression ')'
```

Comment:
Looks like I already used that construct to do the checking. I have changed the code as you suggest.